h2o-core/src/main/java/water/H2ONode.java

```diff
@@ -581,13 +581,20 @@ private boolean keepSending() {
               _outgoingMsgQ.put(bb); // put back and give someone else a chance to deliver
               break; // terminate
             }
-            while( bb != null ) {         // while have an BB to process
-              assert !bb.isDirect() : "Direct BBs already got recycled";
+            while( bb != null) {         // while have an BB to process
```

Comment:
this is seems to be a whitespace-only change and its not improving anything (talking about the while only)

Reply:
Yup, thx